modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc

```diff
@@ -328,9 +328,9 @@ class NDB_Menu_Filter_Form_Resolved_Violations extends NDB_Menu_Filter_Form
         // create user object
         $user        =&        User::singleton();
         $config      =&        NDB_Config::singleton();
-        $study       = $config->getSetting('study');
+        $study       = $config->getSetting('Study');
```

Comment:
@driusan 
It was not working for me with small `s` and worked with capital `S`. But we now know why, so @davidblader will issue a fix.